e2e/testdata/fn-render/fnconfig-outside-package/.expected/config.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exitCode: 1
+stdErr: |
+  Kptfile is invalid:
+  Field: `pipeline.mutators[1].configPath`
+  Value: "some/../../home/.kube/config"
+  Reason: path must not be outside the package directory
```

Comment:
Drop the word "directory". Otherwise, it implies a 1-to-1 correspondence between a package and a directory.

Reply:
Done.